Show port number in failure message

I was trying to build GlassFish 8.0 branch and got a port in use problem that prevented the build from completing.  

```
mvn clean install -Pfastest -T4C
...
[INFO] --- antrun:3.1.0:run (default-manifest) @ glassfish-embedded-all ---
[INFO] Executing tasks
[INFO]     [echo] Generating manifest for /home/smarlow/work/glassfish/appserver/extras/embedded/all/target/classes/META-INF for
[INFO]     [echo]                                    glassfish-embedded-all
[INFO]     [copy] Copying 1 file to /home/smarlow/work/glassfish/appserver/extras/embedded/all/target/classes/META-INF
[INFO] Executed tasks
[INFO]
[INFO] --- hk2-inhabitant-generator:4.0.0-M3:generate-inhabitants (generate-inhabitants) @ glassfish-embedded-all ---
No free port available or it is prohibited.
CLI130: Could not create domain, domain1
Command create-domain failed.
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
   at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:355)
   at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:253)
   at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:884)
   at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:844)
   at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:450)
   at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
   at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
   at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
   at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
   at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
   at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
   at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
   at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
   at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
   at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
   at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:193)
   at org.apache.maven.lifecycle.internal.builder.multithreaded.MultiThreadedBuilder$1.call (MultiThreadedBuilder.java:180)
   at java.util.concurrent.FutureTask.run (FutureTask.java:317)
   at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:572)
   at java.util.concurrent.FutureTask.run (FutureTask.java:317)
   at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1144)
   at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:642)
   at java.lang.Thread.run (Thread.java:1583)
[INFO]
```

Perhaps we should shot the port range.  Will create a pr for this idea.

--- a/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
+++ b/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
@@ -101,6 +101,7 @@ public class CustomTokenClient {
                                 throw new DomainException(format("Given port {0} is not free.", port));
                             }
                         } else {
+                          Integer firstPortTried = port;
                             if (portBase != null && token.getTokenTypeDetails() instanceof PortTypeDetails) {
                                 PortTypeDetails portTypeDetails = (PortTypeDetails) token.getTokenTypeDetails();
                                 port = Integer.parseInt(portBase) + Integer.parseInt(portTypeDetails.getBaseOffset());
@@ -115,7 +116,7 @@ public class CustomTokenClient {
                             // Find next available unused port by incrementing the port value by 1
                             while (!NetUtils.isPortFree(port) && !usedPorts.contains(port)) {
                                 if (port > NetUtils.MAX_PORT) {
-                                    throw new DomainException("No free port available or it is prohibited.");
+                                    throw new DomainException(format("No free port available in range {0} - {1} or it is prohibited.", firstPortTried, port));
                                 }
                                 port++;
                             }

--- a/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
+++ b/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
@@ -116,7 +116,7 @@ public class CustomTokenClient {
                             // Find next available unused port by incrementing the port value by 1
                             while (!NetUtils.isPortFree(port) && !usedPorts.contains(port)) {
                                 if (port > NetUtils.MAX_PORT) {
-                                    throw new DomainException(format("No free port available in range {0} - {1} or it is prohibited.", firstPortTried, port));
+                                    throw new DomainException(format("No free port available in range {0} - {1} or it is prohibited.", firstPortTried, NetUtils.MAX_PORT));
                                 }
                                 port++;
                             }

--- a/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
+++ b/nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java
@@ -101,7 +101,7 @@ public class CustomTokenClient {
                                 throw new DomainException(format("Given port {0} is not free.", port));
                             }
                         } else {
-                          Integer firstPortTried = port;
+                            Integer firstPortTried = port;
                             if (portBase != null && token.getTokenTypeDetails() instanceof PortTypeDetails) {
                                 PortTypeDetails portTypeDetails = (PortTypeDetails) token.getTokenTypeDetails();
                                 port = Integer.parseInt(portBase) + Integer.parseInt(portTypeDetails.getBaseOffset());

